shop/src/utils/useFlattenedShippingZones.js

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from 'react'
+import get from 'lodash/get'
+
+import useShippingZones from './useShippingZones'
+import shippingTimes from './shippingTimes'
+
+const flattenShippingZones = (shippingZones) => {
```

Comment:
nit: add a short comment (jsdoc ideally) explaining what those methods do.

Reply:
Will add it